src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,344 @@
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Enabling tracing data collection can augment your existing errors to answer any question or unknowns about your systems. Tracing also allows you to track critical metrics and service-level objectives (SLO) like throughput, latency, or apdex to build a complete monitoring solution to diagnose and measure your application health.
+
+## Tracing & Distributed Tracing
+
+Applications (for example, web applications) typically consist of interconnected applications/components (also called services). For instance, let us assume a modern web application could be composed of the following components separated by the network boundaries:
+
+- Frontend (Single-Page Application)
+- Backend
+- Background Queue
+- Notification Job
+
+Each of these components may be written in different languages on different platforms. Today, all of these components can be instrumented with the Sentry SDKs to capture error data or crash reports whenever an event occurs in any one of these components.
+
+With tracing, we can follow the journey of the API endpoint requests from their source (for example, from the frontend), and instrument code paths as these requests traverse each component of the application. This journey is called a [**trace**]({%- link _documentation/apm/apm-glossary.md -%}#trace). Traces that cross between components, as in our web application example, are typically called **distributed traces**.
+
+[{% asset apm/tracing-diagram.png alt="Diagram illustrating how a trace is composed of multiple transactions." %}]({% asset apm/tracing-diagram.png @path %})
+
+Each [trace]({%- link _documentation/apm/apm-glossary.md -%}#trace) has a marker called a `trace_id`. Trace IDs are pseudorandom fixed length of alphanumeric character sequences.
+
+By collecting traces of your users as they use your applications, you can begin to reveal some insights such as:
+
+- Investigate and debug what occurred for a specific error event, or issue
+- What conditions are causing bottlenecks or latency issues in the application
+- Which endpoints or operations consume the most time
+
+### Trace Propagation Model
+
+Referring back to our earlier example of the web application consisting of these high-level components:
+
+- Frontend (Single-Page Application)
+- Backend
+- Background Queue
+- Notification Job
+
+A trace [propagates]({%- link _documentation/apm/apm-glossary.md -%}#propagation) first from the frontend, then the backend, and later to the background queue or notification job. Collected [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) from each component are sent back to Sentry asynchronously and independently. Instrumented spans received from one component aren't forwarded to the next component.
```

Comment:
@MimiDumpling would it be a good idea if we mention orphan/stray spans in a sentence in this section? I can provide a brief blurb.

Reply:
@MimiDumpling that works 👍